lib/src/arithmetic/signals/floating_point_logic.dart

```diff
@@ -152,64 +173,88 @@ class FloatingPoint32 extends FloatingPoint {
   /// Construct a 32-bit (single-precision) floating point number
   FloatingPoint32({super.name})
```

Comment:
Should we factor these into other files?

Reply:
yes, i'll do it now

Reply:
we were missing tf32!

Reply:
ack!